web/styles/app_components.css

```diff
@@ -653,7 +653,6 @@ div.overlay {
     }
 
     &:focus {
-        background-color: hsl(240deg 41% 50%);
```

Comment:
Okay.

Reply:
My intention for this change was something different, added this CSS for it:

```css
#send_later {
    /* We want to change background color on hover but do not
       want to do so on focus since focus stays on the button
       when user closes the popover and the compose looks weird
       with one button with different color for no reason. */
    &:focus {
        background-color: hsl(240deg 96% 68%);
    }

    &:hover {
        background-color: hsl(240deg 41% 50%) !important;
    }
}
```